### PR TITLE
Adds Solr Scaler docs

### PR DESCRIPTION
**Pull request by ithesadson**
_Provide a description of what has been changed_
- Adds Solr scaler docs to v2.10

### Checklist

- [x] Commits are signed with Developer Certificate of Origin (DCO)

Releates to:
https://github.com/kedacore/keda/issues/4234
https://github.com/kedacore/keda/pull/4355

### CODE REVIEW

File: content/docs/2.10/scalers/solr.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+      targetQueryValue: "1"
+      activationTargetQueryValue : "3"
+      username: "solr"
+```
+
+**Parameter list:**
+
+- `host` - The hostname for connecting to the Solr service.
+- `query`- A Solr query that should return single numeric value. (Default: `*:*`, Optional)
+- `core` - Your collection name on Solr.
```
Comment (tomkerkhove) [inline content/docs/2.10/scalers/solr.md:29]: I'm not familiar with Solr but wouldn't this be a more reasonable name then?
```suggestion
- `collection` - Your collection name on Solr.
```

Comment (ithesadson) [reply content/docs/2.10/scalers/solr.md]: It is definitely more understandable for the user.

**Review by tomkerkhove — COMMENTED** [review]
LGTM, but waiting for feature to be merged. However, can you move this file to 2.11 please? 2.10 was already shipped

**Comment by ithesadson** [comment]
> LGTM, but waiting for feature to be merged. However, can you move this file to 2.11 please? 2.10 was already shipped

**Comment by ithesadson** [comment]
I closed the comment by mistake, sorry about that.

**Comment by ithesadson** [comment]
I turned off PR by mistake. I turned it back on. Sorry about that.

**Comment by ithesadson** [comment]
I accidentally closed the pull request and then reopened it, sorry about that.

File: content/docs/2.11/scalers/solr.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+```yaml
+triggers:
+  - type: solr
+    metadata:
+      host: "http://solr-service.solr-ns.svc.cluster.local:8983"
+      query: "*:*"
+      collection: "my_core"
+      targetQueryValue: "1"
+      activationTargetQueryValue : "3"
+      username: "solr"
```
[the remark this answers is not in the document]
Comment (ithesadson) [reply content/docs/2.11/scalers/solr.md:22]: I suggest moving username under Trigger Authentication.  Entering it in 2 different places can create confusion for the user.  If you agree, I'll edit it this way.

[the remark this answers is not in the document]
Comment (zroubalik) [reply content/docs/2.11/scalers/solr.md:22]: yeah, please do that :)

**Pull request merged**